src/components/organisms/NewStartProjectPane/NewStartProjectPane.tsx

```diff
@@ -0,0 +1,148 @@
+import {Button} from 'antd';
+
+import {ExclamationCircleOutlined} from '@ant-design/icons';
+
+import styled from 'styled-components';
+
+import CreateFromTemplate from '@assets/CreateFromTemplate.svg';
+import CreateScratch from '@assets/CreateScratch.svg';
+import SelectFolder from '@assets/SelectFolder.svg';
+
+import Colors from '@styles/Colors';
+
+const Container = styled.div`
```

Comment:
Styles should be in a styled.tsx file

Reply:
I will remove it before I set the PR as ready